server-kotlin/src/city_request/CityRequest.kt

```diff
@@ -0,0 +1,26 @@
+package digital.cesko.city_request
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class CityRequest(
+    @JsonProperty("city")
+    val city: String,
+
+    @JsonProperty("email")
+    val email: String,
+
+    @JsonProperty(value = "name", required = false)
+    val name: String,
+
+    @JsonProperty("subscribe")
+    val subscribe: Boolean,
+
+    @JsonProperty("gdpr")
+    val gdpr: Boolean
+) {
+    override fun toString(): String {

```

Comment:
Je nejaky duvod pouzivat jiny toString() nez defaultni z data class? Prijde mi to jako zbytecny boilerplate.